### ISSUE TITLE
It crashed when using clr-installer-gui to install Clear Linux.

### ISSUE THREAD

**Issue by xiangquanliu** [clearlinux/clr-installer#499]
**Describe the bug**
Following the steps of https://docs.01.org/clearlinux/latest/guides/clear/mixer.html to generate the image file. Then use the image to install Clear Linux, it will crash to exit the installation.

**To Reproduce**
1. Following the mixer document to create the OS image which includes the bundle of "clr-installer-gui"
2. Use this image to create the OS installer in USB.
3. Use the USB to boot and install the OS.
4. It will crash after clicking the 'Confirm' button to continue the installation.

**Expected behavior**
Continue the installation until it is completed.

**Screenshots**
![webwxgetmsgimg](https://user-images.githubusercontent.com/24307613/63831578-7ddc0680-c9a1-11e9-9390-6a1a8ff22c8a.jpg)

**Environment (please complete the following information):**
 - Clear Linux OS Version:
The server OS(used to generate the image) version: 30850
 - Installer Version:
2.1.1
 - Platform:
CNL

**Additional context**
Error Trace:
error.Wrap()
        errors/errors.go: 91
swupd.(*SoftwareUpdater).VerifyWithBundles()
        swupd/swupd.go:360

Other information:
I used the older version to do this and no errors found before.

**Comment by xiangquanliu** [comment]
@mdhorn, Thanks for your quick reply!
Do you mean to use HTTPS in builder.conf? If yes, I think the mixer document should be updated accordingly.

_Edit builder.conf to set the value of CONTENTURL and VERSIONURL to the IP address of the nginx* server you set up in the prerequisite Set up a nginx web server for mixer. For example:_

_CONTENTURL="http://192.168.25.52"
VERSIONURL="http://192.168.25.52"_

**Comment by xiangquanliu** [comment]
After being changed from HTTP to HTTPS, the  image can't be installed as the check of  "Test Network Settings" can't pass. But the server URL can be accessed by IE/Chrome. So do you have any comments about this? Thanks!

**Comment by mdhorn** [comment]
@xiangquanliu 
Is the webserver sharing the mix content properly configured to share via HTTPS? Does it have a valid certificate that signed by a source the OS recognizes? 
`curl https://192.168.25.52/<yourMIX>/version/latest_version`

**Comment by xiangquanliu** [comment]
@mdhorn 
Do you have any document about this for Clear Linux OS? Thanks!

**Comment by mdhorn** [comment]
Until we can implement a change to the clr-installer, you can try working around this issue by creating a local swupd configuration file which allows the use of HTTP

cat /etc/swupd/config
```
[GLOBAL]
# Allow updates over insecure connections
allow_insecure_http=true
```

This needs to be in the image file you create.
You should be able to automate this by using a post-install hook.

**Comment by xiangquanliu** [comment]
![Untitled](https://user-images.githubusercontent.com/24307613/64393354-92945a80-d083-11e9-876e-f0b7ab1a81af.png)

@mdhorn , please see the log above. The "network check" didn't pass even change it to http.
But as you can see, it can pass if you run the the curl command manually. I don't know why it failed in clr-installer.

**Comment by xiangquanliu** [comment]
I tried to update "mixer/update/image/10/full/usr/share/defaults/swupd/config" to change this option "allow_insecure_http=true" and build the image.  Then tried to install the the image using http and it still failed.

**Comment by mdhorn** [comment]
We will be adding an `--insecure` flag to the installer similar to the one for swupd to allow HTTP on the command line as well as automatically creating the /etc/swupd/config file with the option.

**Comment by mdhorn** [comment]
This was implemented in PR #552 
and should be in the next release

**Comment by mdhorn** [comment]
This was fixed in clr-installer release [2.3.0](https://github.com/clearlinux/clr-installer/releases/tag/2.3.0) and first available in Clear Linux OS release 31140 or later.
